libs/networking/cohttp_lwt_unix_with_ctx/dune

```diff
@@ -0,0 +1,21 @@
+(library
```

Comment:
we might want to consolidate all those libs in just libs/networking/
Right now we have 3 subdirs with each one file ... what about just one libs/networking/ with those 3 files?

Reply:
We wouldn't want to condense all of them, at least this one should be separate since it depends on cohttp-lwt-unix, and merging all 3 would mean lsp.js now depends on unix stuff

Reply:
ok, would be nice to add comments in the dune files of why we split libs/networking with this very small granularity